config/config.toml.example

```diff
@@ -367,3 +367,10 @@ history-size = 24
 [experimental]
 # enable column attribute `auto_random` to be defined on the primary key column.
 allow-auto-random = false
+
+# server level isolation read by engines and labels
+[isolation-read]
+# engines means allow the tidb server read data from which types of engines. options: "tikv", "tiflash", "tidb".
+engines = ["tikv", "tiflash", "tidb"]
+# labels means allow the tidb server read data from which labels. Empty set means all labels are allowed.
```

Comment:
Could you explain more on the purpose of `labels` in the comment? or the use cases of the `labels`?